### DISCUSSION TITLE
Scores not submitting sometimes

### DISCUSSION THREAD

**Discussion by KnuffKirby** [ppy/osu#30823] [Q&A]
### Type

Game behaviour

### Bug description

Me aswell as other people have noticed, that sometimes scores dont submit in lazer aswell as an error message dropping when this happens. In my case, it happened because the game logged me out during my play, but I have also had it happen without logging out.

### Screenshots or videos

![osu_2024-11-21_19-33-17](https://github.com/user-attachments/assets/57d5421a-7165-49b8-ae2d-174258088bb0)

### Version

2024.1115.3-lazer

### Logs

[Uploading compressed-logs.zip…]()

**Comment by bdach** [comment]
Please upload the logs again, you posted the issue before they actually fully uploaded.

**Comment by KnuffKirby** [comment]
[compressed-logs.zip](https://github.com/user-attachments/files/17852261/compressed-logs.zip)

**Comment by smoogipoo** [comment]
Your connection is disconnecting intermittently for some reason.
- If you're using IPv6, try turning it off and using only IPv4.
- If you're using a proxy, disable it.
- Otherwise, it's going to be something else. Your ISP or routing elsewhere going wrong.

Maybe @ThePooN could check if there's been any spike in CloudFlare errors from Germany, but I highly doubt this is going to be CF's fault.
